src/main/kotlin/no/nav/familie/ks/sak/kjerne/fagsak/FagsakService.kt

```diff
@@ -254,6 +258,15 @@ class FagsakService(
         }
     }
 
+    internal fun settEgenAnsattStatusPåFagsakDeltagere(fagsakDeltagere: MutableList<FagsakDeltagerResponsDto>): List<FagsakDeltagerResponsDto> {
```

Comment:
Trenger denne å være `mutable`? 

```suggestion
    internal fun settEgenAnsattStatusPåFagsakDeltagere(fagsakDeltagere: List<FagsakDeltagerResponsDto>): List<FagsakDeltagerResponsDto> {
```

Reply:
Nop, ble generert slik. Fikset ✅